### DISCUSSION TITLE
Wifi credentials are retained after SD card format

### DISCUSSION THREAD

**Discussion by mightymos** [OnionUI/Onion#1660] [Support] [answered]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.3.1-1

### Miyoo Firmware Version

202306282128

### Provide a clear and concise description of the issue

I am selling my Miyoo Mini Plus and formatted the SD card.
I then copied over a fresh install of OnionOS and Ports.

I also factory reset the stock firmware without SD card installed.
My wifi continues to connect even after SD card was wiped even though I did not reenter credentials.

### Steps to Reproduce

1. Have an existing OnionOS install with wifi credentials
2. Remove SD card and factory reset stock
3. Format SD card to FAT32
4. Copy over fresh install of OnionOS and Ports to SD card
5. Wifi credentials are still retained

### Provide any additional context or information that may help us investigate your issue

None at this time.

**Comment by Aemiii91** [comment, marked as answer]
https://onionui.github.io/docs/apps/tweaks#reset-mainui-settings
